source/OutfitInfoDisplay.cpp

```diff
@@ -462,28 +462,28 @@ void OutfitInfoDisplay::UpdateAttributes(const Outfit &outfit)
 		attributeLabels.emplace_back("ammo:");
 		attributeValues.emplace_back(outfit.Ammo()->DisplayName());
 		attributesHeight += 20;
-		if(outfit.AmmoUsage() != 1)
+		if(weapon.AmmoUsage() != 1)
```

Comment:
Introducing something like `AmmoConsumption` feels a bit too much for this PR, so I moved the ammo pair into weapon completely. But still left an accessor function in outfit to avoid making the code confusing when it comes to an outfit storing ammo or a weapon consuming ammo